src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -667,11 +672,26 @@
 helps['network asg'] = """
     type: group
     short-summary: Manage application security groups.
```

Comment:
Include `ASG` acronym in the short description if it is used in the long description.

Reply:
Addressed.